website/docs/components/form/layout/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Form Layout
+description: Form layout components are used to create consistent structure and spacing for form elements.
+caption: Components used to create consistent structure and spacing for form elements.
+links:
+  figma: >-
+    https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/branch/azcdumnXvCyGfSVfEVZhGH/HDS-Components-v2.0?node-id=83107-2070&p=f&t=CeumEO6vBV6h0IuD-0
+  github: >-
+    https://github.com/hashicorp/design-system/tree/main/packages/components/src/components/hds/form
+related:
+  - patterns/form-patterns
+previewImage: assets/illustrations/components/form/layout.jpg
+navigation:
+  label: Layout
```

Comment:
[question] Would changing the label to "Form Layout" also change the name on the card in the Overview page? Out of context, this is a bit odd, but I'm not sure what controls this. 

<img width="1240" alt="Screenshot 2025-06-24 at 4 13 36 PM" src="https://github.com/user-attachments/assets/b4f14610-5382-4263-ac5e-5714ad263bee" />

Reply:
Yes, in this case is similar to "Primitives" next to it. But it's a good point, we could consider to use the other title, for the cards (maybe @KristinLBradley can you create a ticket for this, to do as follow-up when we have time?)

Reply:
@didoo Do you mean have the cards and sidebar nav item use the "title" instead of the navigation "label" or am I misunderstanding?

Reply:
No, to have the cards to use the `title` instead of the `navigation.label`. If this doesn't work because we have other exceptions, we could split the `previewImage` and have `preview.image` and `preview.label`.

Reply:
Resolving to clean up